src/jvm/test/resources/expected_logs/run_concurrent_test/main_thread_blocked_jdk11.txt

```diff
@@ -30,15 +29,9 @@ Detailed trace:
 |               compareAndSetState(0, 1): true at ReentrantLock$Sync.nonfairTryAcquire(ReentrantLock.java:130)                                       |
 |                 state.compareAndSet(0, 1): true at AbstractQueuedSynchronizer.compareAndSetState(AbstractQueuedSynchronizer.java:612)              |
 |       ConditionObject#1.await() at ArrayBlockingQueue.take(ArrayBlockingQueue.java:417)                                                            |
-|         /* The following events repeat infinitely: */                                                                                              |
-|     ┌╶> LockSupport.park(ConditionObject#1) at AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2081)              |
-|     |     LockSupport.setBlocker(Thread#1, ConditionObject#1) at LockSupport.park(LockSupport.java:193)                                            |
-|     |       Unsafe#1.putObject(Thread#1, 76, ConditionObject#1) at LockSupport.setBlocker(LockSupport.java:144)                                    |
-|     |     PARK at LockSupport.park(LockSupport.java:194)                                                                                           |
-|     |     LockSupport.setBlocker(Thread#1, null) at LockSupport.park(LockSupport.java:195)                                                         |
-|     |       Unsafe#1.putObject(Thread#1, 76, null) at LockSupport.setBlocker(LockSupport.java:144)                                                 |
-|     |   NonfairSync#1.isOnSyncQueue(Node#1): false at AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2080)       |
-|     |     node.waitStatus ➜ -2 at AbstractQueuedSynchronizer.isOnSyncQueue(AbstractQueuedSynchronizer.java:1673)                                   |
-|     └╶╶ switch (reason: active lock detected)                                                                                                      |
+|         LockSupport.park(ConditionObject#1) at AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2081)              |
```

Comment:
Consider creating an issue to not show `LockSupport` internals

Reply:
https://github.com/JetBrains/lincheck/issues/627